cli/flox-activations/src/main.rs

```diff
@@ -1,18 +1,23 @@
 use clap::Parser;
 use cli::Cli;
+use xdg::BaseDirectories;
 
 mod activations;
 mod cli;
 
-type Error = anyhow::Error;
+pub type Error = anyhow::Error;
 
 fn main() -> Result<(), Error> {
     let args = Cli::parse();
+
+    let dirs = BaseDirectories::with_prefix("flox")?;
+    let cache_dir = dirs.get_cache_home();
```

Comment:
blocking: I think for testing to work in the future we need to thread this all the way down from `flox` somehow. Might need to add a `--cache-dir` arg?

Reply:
i left it at --cache-dir now, but if the logic to compute the runtime dir isnt required to be in activations i'd rather like use to replace that with a required arg and thread that down from flox considering flox needs the directory as well to read activations as does the watchdog

Reply:
Yeah passing `runtime-dir` would work too